setup.py

```diff
@@ -87,9 +87,9 @@ def read(path):
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
```

Comment:
Should we then also add 3.7 to the Travis CI configuration?
https://github.com/crate/crash/pull/303/files#diff-354f30a63fb0907d4ad57269548329e3R5

If so, I would also mention it in the changelog entry.